docs/Committers.md

```diff
@@ -24,6 +24,9 @@ vote of all members of the Community Governance Committee.
 The running minutes for the Community Governance Community meetings are located
 at [aka.ms/openenclave/cgc](https://aka.ms/openenclave/cgc).
 
+The Committee can be reached by sending email to
+oesdkcgc@lists.confidentialcomputing.io.
```

Comment:
Folks can actually send a private mail to the administrators of the oesdk mailiing list, perhaps that's a better way to do this for anything that is not just sent to oesdk as a public mail?

Reply:
Are the administrators of that list enumerated somewhere? If that isn't a publicly-visible artefact of the list itself, are you concerned it may get out of date if the list membership is updated?

Reply:
This raises a different question as I think about it -

is the intent of the oesdkcgc@ list to be a private discussion space for CGC members, or a public discussion space _about_ community governance?

If the former, it should remain private and moderate any non-member posts. If the latter, it should be publicly visible, and allow posts and subscriptions from any member of the oesdk@ community.

Reply:
It is intended to be the former.

But should others be able to send messages to the GCG that they wish to be private (for example CoC violations/concerns)? Or do you think it should just be a separate list as @CodeMonkeyLeet  suggests?